src/hotspot/share/gc/shenandoah/shenandoahMonitoringSupport.hpp

```diff
@@ -26,6 +26,7 @@
 #define SHARE_GC_SHENANDOAH_SHENANDOAHMONITORINGSUPPORT_HPP
 
 #include "memory/allocation.hpp"
+#include "gc/shared/collectorCounters.hpp"
```

Comment:
If this is required, could it be made in its own PR? It's odd to see this PR changes files exclusively owned by other collectors.